ocha_ai_chat.install

```diff
@@ -44,7 +44,6 @@ function ocha_ai_chat_schema() {
         'type' => 'text',
         'size' => 'medium',
         'not null' => TRUE,
-        'default' => '',
```

Comment:
Why the removal of the default value here and below?

Reply:
Fixed, switched to MariaDb 10.6